config/sync/pathauto.pattern.news_url.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 078f1f3d-6d95-47cb-830e-71690095790d
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: news_url
+label: 'News URL'
+type: 'canonical_entities:node'
+pattern: '[current-date:custom:Y\/m\/d]/[node:title]'
```

Comment:
I believe this one should be `[node:created:custom:Y\/m\/d]/[node:title]`

Reply:
updated path name